spec/models/dialog_field_association_validator_spec.rb

```diff
@@ -12,9 +12,12 @@
 
     context "when the associations are not blank" do
       context "when there are no circular references" do
-        it "returns false" do
+        it "returns false on the trivial case" do
           expect(dialog_field_association_validator.circular_references("foo" => ["baz"])).to eq(false)
-          expect(dialog_field_association_validator.circular_references("foo" => %w(foo2 foo4), "foo2" => ["foo3"], "foo3" => ["foo4"])).to eq(false)
+        end
+
+        it "returns false on the non-trivial case" do
```

Comment:
Wow, this method returns `false` or an array? :cry:

Reply:
Okay, so I'd be happy to change this. But essentially the change is just adding a helper method that checks to see if the array has nothing in it. I don't like the fact that the only one of these comments of yours I know how to fix is the stupid trivial one.